src/solver/hydro/management/management.cpp

```diff
@@ -350,13 +344,14 @@ bool HydroManagement::checkMinGeneration(uint numSpace)
         uint z = area.index;
         const auto& ptchro = NumeroChroniquesTireesParPays[numSpace][z];
         auto tsIndex = (uint)ptchro.Hydraulique;
+        auto tsIndexPowerCredits = ptchro.HydrauliquePowerCredits;
 
         bool useHeuristicTarget = area.hydro.useHeuristicTarget;
         bool followLoadModulations = area.hydro.followLoadModulations;
         bool reservoirManagement = area.hydro.reservoirManagement;
 
         if (!reservoirManagement)
```

Comment:
The condition "if (!reservoirManagement)" should be removed and "checkHourlyMinMaxGeneration" called in all cases.
This is wrong legacy from CR22 - minimum hydro storage generation!
Agree?